src/click/termui.py

```diff
@@ -46,9 +46,9 @@
 
 
 def hidden_prompt_func(prompt):
-    import getpass
+    import stdiomask
```

Comment:
Unfortunately, we can't add external dependencies.

Note that if you did add a dependency, you'd want to add it to `install_requires` as well.